lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -183,6 +176,42 @@ class AwsCompileFunctions {
 
       _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
         newOutputObject);
+
+      if (typeof this.serverless.service.provider.iamRoleARN !== 'string') { // only if default IAM
+        const logRetentionInDays = 14; // TODO make configurable?
+        const logGroupTemplate = `
+          {
+            "${normalizedFunctionName}LogGroup": {
+              "Type" : "AWS::Logs::LogGroup",
+              "Properties" : {
+                "LogGroupName" : "/aws/lambda/${FunctionName}",
+                "RetentionInDays" : "${logRetentionInDays}"
```

Comment:
Good call.  It wasn't clear if it was an optional attribute given that no default value is specified.  A problem with not being able to do this in the native fashion.  Maybe mostly a problem with my laziness.